appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java

```diff
@@ -95,8 +95,8 @@ private Index getIndex() {
     } catch (SearchException e) {
       // handle exception...
```

Comment:
The comment says "handle exception", and it is deferred to users how to handle it :)